types/ApiRequests.ts

```diff
@@ -116,6 +125,10 @@ export interface AttendEventRequest {
   asStaff?: boolean;
 }
 
+export interface AddEventFeedbackRequest {
+  feedback: string[];
```

Comment:
![image](https://user-images.githubusercontent.com/33337209/96164074-91ad3980-0ecf-11eb-8501-6d49cb040fba.png)

judging from this it seems like its short comments / emojis as a reaction to the event. not sure what the custom feedback will allow for so I'll ask Design about that

Reply:
re: custom feedback- design says that it can be a few sentences for custom feedback. Also note that design wants us to reward users points (probably 2 pts) for this event feedback. I think leaving feedback as a string array is still fine though.

Reply:
We also need to add feedback as an activity then. I'll try wrapping up the enums-to-strings PR soon so you don't need to add a migration for the new ActivityType, just need to clean up all the changes I made getting it to work.